front-end/src/pages/AuthorPage.tsx

```diff
@@ -90,7 +91,7 @@ export default function AuthorPage(props: Props) {
             </CardBody>
           </Card>
         </Col>
-        {<PostList postEntries={postEntries} setPostEntries={setPostEntries} loggedInUser={props.loggedInUser} />}
+        {author && displayPosts()}
```

Comment:
I like the use of this conditional

Reply:
ty, fixed/added viewing other authors public stream. 

the only issue is that if we're not logged in and we try to view an author's profile, there is an authentication issue that I don't know where its coming from (probably on the backend) so it wont GET the author's posts. for now, I have it so that posts only get displayed if you're logged in and viewing an author's profile